apps/agent/contracts/standards/IERC1155Receiver.sol

```diff
@@ -0,0 +1,35 @@
+pragma solidity 0.4.24;
+
+
+// See https://eips.ethereum.org/EIPS/eip-1155
+interface IERC1155Receiver {
+    /**
+    * @notice Handle the receipt of a single ERC1155 token type.
+    * @dev An ERC1155-compliant smart contract MUST call this function on the token recipient contract, at the end of a `safeTransferFrom` after the balance has been updated.
+    *      This function MUST return `bytes4(keccak256("onERC1155Received(address,address,uint256,uint256,bytes)"))` (i.e. 0xf23a6e61) if it accepts the transfer.
+    *      This function MUST revert if it rejects the transfer.
+    *      Return of any other value than the prescribed keccak256 generated value MUST result in the transaction being reverted by the caller.
+    * @param _operator The address which initiated the transfer (i.e. msg.sender)
+    * @param _from The address which previously owned the token
+    * @param _id The ID of the token being transferred
```

Comment:
In many places, the EIP refers to this field as "token type", not always tho, but I think it is clearer that way

```suggestion
    * @param _id The ID of the token type being transferred
```

Reply:
Agreed, but these were the comment strings in the finalized EIP so I'd prefer to leave it as-is :).